File: src/main/java/io/github/electricpoultry/andonium/util/registers/Registers.java
package io.github.electricpoultry.andonium.util.registers;

import io.github.electricpoultry.andonium.block.ModBlocks;
import io.github.electricpoultry.andonium.item.ModCreativeModeTabs;
import io.github.electricpoultry.andonium.item.ModItems;
import net.neoforged.bus.api.IEventBus;

public class Registers {
    public static void registerAll(IEventBus eventBus) {
        ModItems.register(eventBus);
        ModBlocks.register(eventBus);
        ModCreativeModeTabs.register(eventBus);
    }
}

File: src/main/java/io/github/electricpoultry/andonium/util/registers/Registers.kt
package io.github.electricpoultry.andonium.util.registers

import io.github.electricpoultry.andonium.item.ModCreativeModeTabs
import io.github.electricpoultry.andonium.block.ModBlocks
import io.github.electricpoultry.andonium.item.ModItems
import net.neoforged.bus.api.IEventBus

object Registers {
    @JvmStatic
    fun registerAll(eventBus: IEventBus) {
        ModItems.register(eventBus)
        ModBlocks.register(eventBus)
        ModCreativeModeTabs.register(eventBus)
    }
}
